frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/contextmenu/OverrideForkPointOfSelectedBranchAction.java

```diff
@@ -0,0 +1,16 @@
+package com.virtuslab.gitmachete.frontend.actions.contextmenu;
+
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import io.vavr.control.Option;
+
+import com.virtuslab.gitmachete.frontend.actions.base.BaseOverrideForkPointAction;
+import com.virtuslab.gitmachete.frontend.actions.expectedkeys.IExpectsKeySelectedBranchName;
+
+public class OverrideForkPointOfSelectedBranchAction extends BaseOverrideForkPointAction
```

Comment:
Hmmm this is the sole extending class... do we plan for other extending classes in the near future (as in, to be introduced within the scope of this task, or e.g. by #427)?

Reply:
Actually, the same point/question applies to other single-subclass base actions classes

Reply:
I don't think so. We could remove ones unrelated to with #427. WDYT?

Reply:
On the other hand, even though we do not provide a toolbar button for fork point override it would be nice to be able to invoke it via `Find Action...`

Reply:
Ok, let's address it on #427